src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -601,12 +601,24 @@
     text: az sql midb list -g mygroup --mi myinstance
 """
 
+helps['sql midb list-deleted'] = """
+type: command
+short-summary: List restorable deleted managed databases.
+examples:
+  - name: List all restorable deleted managed databases on Managed Instance .
+    text: az sql midb list-deleted -g mygroup --mi myinstance
+"""
+
 helps['sql midb restore'] = """
 type: command
 short-summary: Restore a managed database.
 examples:
-  - name: Restore a managed database using Point in time restore
+  - name: Restore a live managed database using Point in time restore
     text: az sql midb restore -g mygroup --mi myinstance -n mymanageddb --dest-name targetmidb --time "2018-05-20T05:34:22"
+  - name: Restore a dropped managed database using Point in time restore
+    text: az sql midb restore -g mygroup --mi myinstance -n mymanageddb --dest-name targetmidb --time "2018-05-20T05:34:22" --deleted-time "2018-05-20T05:34:22"
```

Comment:
What is the difference for --time and --deleted-time?

Reply:
--deleted-time is required to identify dropped database, because primary key to identify dropped database is database name + drop time. For live databases name is good enough, because you cannot have multiple databases with the same name. --time on other had represents the point we are restoring to.